### PR TITLE
Feature/circle editor

### PR DESCRIPTION
**Pull request by tomermoshe**

### CODE REVIEW

File: src/angular-cesium-entities-editor/components/circles-editor/circles-editor.component.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<ac-layer #editPointsLayer acFor="let point of editPoints$" [context]="this" [zIndex]="1">
+    <ac-point-desc props="{
+        position: point.getPosition(),
+        pixelSize: getPointSize(point),
+        color: Cesium.Color.WHITE,
```
Comment (eitanfr) [inline src/angular-cesium-entities-editor/components/circles-editor/circles-editor.component.html]: get color and other props from config options,
check out what I did with polygons after i will enter my PR

Comment (eitanfr) [reply src/angular-cesium-entities-editor/components/circles-editor/circles-editor.component.html]: if you think you dont have enough time for it we can consider doing it in another task...

File: src/angular-cesium-entities-editor/models/editable-circle.ts [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+    if (!this.doneCreation) {
+      return;
+    }
+
+    const radius = this.getRadius();
+    this.center.setPosition(newCenterPos);
+    this.radiusPoint.setPosition(
+      GeoUtilsService.pointByLocationDistanceAndAzimuth(this.getCenter(), radius, Math.PI / 2, true));
+
+    // const delta = GeoUtilsService.getPositionsDelta(this._center.getPosition(), newCenterPos);
```
Comment (eitanfr) [inline src/angular-cesium-entities-editor/models/editable-circle.ts]: remove

File: src/angular-cesium/models/vec3.ts [hunk 3]
```diff
@@ -0,0 +1,5 @@
+export interface Vec3 {
```
Comment (eitanfr) [inline src/angular-cesium/models/vec3.ts]: what the difference between vec3 and cartesian3? 
cant we use cartesian3?

File: CHANGELOG.md [hunk 4]
```diff
@@ -5,7 +5,9 @@
 
 ### Features
 * Added `selectionManagerService` and example in selection-layer.component.ts
-* Added context menu service that allows dynamic injection of custom components.  
+* Added context menu service that allows dynamic injection of custom components.
+* Added Circle editor `CirclesEditorService` - allow for creating circles and editing them from the map of from the code.
+* edited polygons are now draggable.
```
Comment (eitanfr) [inline CHANGELOG.md]: no breaking changes right?

Comment (tomermoshe) [reply CHANGELOG.md]: nope

File: src/angular-cesium/models/vec3.ts (hunk 3 above)
Comment (tomermoshe) [reply src/angular-cesium/models/vec3.ts]: its not the same. cartesian3 must have clone() + equals()

File: src/angular-cesium-entities-editor/components/circles-editor/circles-editor.component.html (hunk 1 above)
Comment (tomermoshe) [reply src/angular-cesium-entities-editor/components/circles-editor/circles-editor.component.html]: id rather move to dynamic layer. another task

**Pull request merged**